src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -227,39 +230,9 @@ def add_devices_to_nwbfile(nwbfile: NWBFile, metadata: dict | None = None) -> NW
     return nwbfile
 
 
-def _create_imaging_plane_from_metadata(nwbfile: NWBFile, imaging_plane_metadata: dict) -> ImagingPlane:
```

Comment:
Now that are settings defaults having this extra private function obscures the logic so I am removing it.